docs/blog/2018-02-26-documentation-project/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Gatsby’s Documentation Project
+author: Shannon Soper
+date: 2018-02-22
+---
+
+Generally, people really like Gatsby docs and Gatsby in general. In fact, we’ve talked about tracking the number of curse words and endearingly pejorative terms that people use when tweeting about Gatsby. Here’s a sampling:
+
+>”HOLY SHIT @gatsbyjs && @NetflifyCMS where have you been all my life”
+
+>”The sites it produces are *disgustingly fast*.”
+
+>”I mean @gatsbyjs has been on my list to learn for many months, and everyone praises it, but OMFG what a straightforward platform!!! Very impressed”
+
+> “We just released our marketing site built with @gatsbyjs. It’s stupid fast, and best of all our marketing folks don’t have to bother us developers about changes anymore :D simplyagree.com.”
+
+And that’s just how much people like Gatsby in general. Now let’s look at how much people like the _docs_:
+
+## How much people like the docs :D
+
+In over 60 interviews with Gatsby users, I asked “What do you like about Gatsby?” Documentation comes out as one of the clear winners, as you can see in this list of the top ten things interviewees like about Gatsby, courtesy of Unbird.com analytics:
+
+![Unbird.com analytics screenshot](unbird-gatsby-likes.png)
+
+To dive a little deeper, here are some quotes about how much people like the docs and tutorials:
+
+> “Very advanced docs & tutorials for this stage in a project.”
+> “Documentation is wonderful.”
+> “For me personally, the most important thing to help me reach my goals with Gatsby is good documentation with examples - which is, in my opinion, already the case compared to other projects.”
+> “Tutorials are very beginner friendly and great for someone who wants to become a better developer.”
+> “The tutorials are very easy to understand and go through great lengths to explain every line of code necessary.”
+
```

Comment:
There's some great tweets praising the tutorials/docs as well

Reply:
That's true. Would those be better than these quotes?

Reply:
I'd mix both in.